doc/interfaces/BUAN_flow.rst

```diff
@@ -0,0 +1,85 @@
+.. _BUAN_flow:
+
+=====================================================================================
+   Tutorial for BUndle ANalytics (BUAN) framework using DIPY Workflow command lines
+=====================================================================================
+
+This tutorial walks through the steps for reproducing Bundle Analytics results
+on Parkinson's Progression Markers Initiative (PPMI) data derivatives.
+Bundle Analytics is a framework for comparing bundle profiles and shapes of
+different groups. In this example, we will be comparing healthy controls and
+patients with parkinson's disease. We will be using PPMI data derivatives generated
+using DIPY.
+
+
+First we need to download streamline atlas of 30 bundles in MNI space from::
+
+    https://figshare.com/articles/Atlas_of_30_Human_Brain_Bundles_in_MNI_space/12089652
+
+For this tutorial we will be using a test sample of DIPY Processed Parkinson's
+Progression Markers Initiative (PPMI) Data Derivatives. It can be downloaded
+from the link below::
+
+     https://doi.org/10.35092/yhjc.12098397
+
+There are two parts of Bundle Analytics group comparison framework,
+bundle profile analysis and bundle shape similarity.
+
+-------------------------------------------------------------------
+      We first start with group comparison of bundle profiles
+-------------------------------------------------------------------
+
+For generating bundle profile data (saved as .h5 files):
+You must have downloaded bundles folder of 30 atlas bundles and subjects folder
+with PPMI data derivatives.
+
+NOTE: Make sure all output folders are empty and do not get overridden.
+Following workflows require specific input directory structure.
+
+Create an out_dir folder (eg: bundle_profiles)::
+
+    mkdir bundle_profiles
```

Comment:
Can you add `$` in front. e.g `$ mkdir bundle_profiles`. Same comment for all command line below

Reply:
with $ it generates something like this:

:math:` mkdir lmm_plots

Reply:
with $
` `:math:`mkdir bundle_profiles `

without $
`mkdir bundle_profiles`

Reply:
<img width="836" alt="Screen Shot 2020-04-15 at 5 42 13 PM" src="https://user-images.githubusercontent.com/26060471/79392785-f3e9a980-7f41-11ea-85fd-afc286830c03.png">

Reply:
ok, I see, thank you for this info